### PR TITLE
Multiplier plugin

### PR DESCRIPTION
**Pull request by Alessar**
### Required for all PRs:

- [x] Signed [CLA](https://influxdata.com/community/cla/).
- [x] Associated README.md updated.
- [x] Has appropriate unit tests.

### CODE REVIEW

File: plugins/processors/multiplier/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Multiplier Processor
+
+The multiplier processor is used to change the values of metrics with some multiplication factor.
+
+### Examples:
+
+* [input.system](./plugins/inputs/system) plugin produces metric [uptime](./plugins/inputs/system/SYSTEM_README.md#metrics) with amount of seconds after computer start. If it is needed to get value in hours multiplier configuration should contain
+"system uptime=0.00027777777"
+
+* Some **metric** contains **value** in range (0;&nbsp;1). If it is needed to get values in more human readable percentage format multiplier configuration should contain
```
Comment (igprya) [inline plugins/processors/multiplier/README.md]: I assume that the range is supposed to be `[0; 1]`, unless 0 and 1 are indeed excluded. 🙂

File: plugins/processors/multiplier/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+"metric value=100"
+
+**Note:** Types of changed values stay original.
+
```
Comment (igprya) [inline plugins/processors/multiplier/README.md]: It's a matter of taste for the most part, but I would've phrased these two notes this way:
`Note: Multiplied values keep thier original types.`
`Note: Values that cannot be multiplied remain unchanged.`

File: plugins/processors/multiplier/README.md (hunk 1 above)
Comment (Alessar) [reply plugins/processors/multiplier/README.md]: Thanks. Fixed.

File: plugins/processors/multiplier/README.md (hunk 2 above)
Comment (Alessar) [reply plugins/processors/multiplier/README.md]: Thanks. Fixed.

**Comment by Alessar** [comment]
@danielnelson Processor plugin is done and is waiting to be merged.
For future changes it is good to get changes discribed in #3276 issue.

**Comment by igprya** [comment]
@danielnelson Hi Daniel! Are there any plans to merge this PR anytime soon, by any chance? :slightly_smiling_face:

**Comment by octopotato** [comment]
@danielnelson any update on when, if, this plugin will be merged?  We'd like to leverage this plugin in our service and our fix so far has been to apply this patch locally and we'd like to move away from that

**Comment by danielnelson** [comment]
Actually going to go with another pull request that utilizes Starlark and provides this functionality: https://github.com/influxdata/telegraf/pull/7660.

The other PR has an example of multiplying two fields: https://github.com/influxdata/telegraf/pull/7660/files#diff-72cc07fe3b959a05c0fa8adbfb809c1f

@Alessar Sorry about leaving this pull request for so long, but I'm going to close in favor of #7660.

**Comment by Alessar** [comment]
@danielnelson looks like `Starlark` is really very powerful plugin if it uses programming language.
`Multiplier` plugin is simpler. But in any case, thanks that I had a chance to attempt to contribute.

@octopotato thanks that you found my plugin and used it. I am glad that it was useful.

Maybe in some future, somebody wants that `Multiplier` rises like a Phenix - feel free to reopen #4380.
And if I need to change something - feel free to ping me.

**Pull request closed without merging**